include/picongpu/fields/currentDeposition/EmZ/EmZ.hpp

```diff
@@ -45,7 +45,8 @@ namespace picongpu
 
             PMACC_CASSERT_MSG(
                 __EmZ_supercell_or_number_of_guard_supercells_is_too_small_for_stencil,
-                pmacc::math::CT::min<typename pmacc::math::CT::mul<SuperCellSize, GuardSize>::type>::type::value
+                sizeof(T_ParticleShape) // defer assert evaluation
```

Comment:
Shouldn't we use `T_ParticleShape*` here to not depend on the class definition being available

Reply:
I also thought about it, I will use the ptr.